drivers/nrf_802154_serialization/src/platform/zephyr/nrf_802154_spinel_backend_ipc.c

```diff
@@ -17,14 +17,20 @@
 #define LOG_MODULE_NAME spinel_ipc_backend
 LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
+#define IPC_MASTER IS_ENABLED(CONFIG_RPMSG_SERVICE_MODE_MASTER)
+
 static K_SEM_DEFINE(ready_sem, 0, 1);
 static int endpoint_id;
 
+#if IPC_MASTER
+
 static bool endpoint_is_bound(void)
 {
 	return rpmsg_service_endpoint_is_bound(endpoint_id);
```

Comment:
Why does `rpmsg_service_endpoint_is_bound()` have to be wrapped here instead of calling it directly (L:53)?
Does `rpmsg_service_endpoint_is_bound()` block forever?

Reply:
Yes, it be called directly. I will remove this function. `rpmsg_service_endpoint_is_bound` is non-blocking.

Reply:
okay, thanks for reply. Can you please open a PR request that contains only the fix in this file? We need it to resolve CI issue:
```
/home/work/modules/hal/nordic/drivers/nrf_802154_serialization/src/platform/zephyr/nrf_802154_spinel_backend_ipc.c:23:13: error: 'endpoint_is_bound' defined but not used [-Werror=unused-function]
   23 | static bool endpoint_is_bound(void)
      |             ^~~~~~~~~~~~~~~~~
```
See also https://github.com/nrfconnect/sdk-nrf/pull/3715.